apps/web/src/app/[lang]/@aside/(.)profile/edit/loading.tsx

```
@@ -0,0 +1,25 @@
+        avatar=""
+        name=""
+        surname=""
+        id={null}
+        nickname=""
+        closeUrl=""
+        description=""
+        leadImageUrl=""
+        isLoading={true}
+        useEditProfile={useEditProfile}
+        useUploadThingFileUploader={useUploadThingFileUploader}
+        successfulEditCallback={() => {}}
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)